### PR TITLE
rework the tests, add an import test, test on python version 7 and 8 …

### PR DESCRIPTION
**Pull request by TAG-Epic**
…aswell

### CODE REVIEW

**Comment by TAG-Epic** [comment]
Any update on this?

File: run_tests.py [hunk 1]
```diff
@@ -29,17 +28,32 @@ def test_pylint():
...
-    if flake_failed or pylint_failed:
-        sys.exit(1)
+
+if __name__ == '__main__':
+    tests = [test_flake8, test_pylint, test_import]
+    fail_count = 0
+
+    for test in tests:
+        test_name = test.__name__
+        formatted_test_name = f"-- {test_name[5:]} test --"
```
Comment (devoxin) [inline run_tests.py]: Single quotes

File: run_tests.py [hunk 2]
```diff
@@ -29,17 +28,32 @@ def test_pylint():
...
+        print(formatted_test_name)
+
+        failed = test()
+        if failed:
+            print("Failed")
```
Comment (devoxin) [inline run_tests.py]: Single quotes

File: run_tests.py [hunk 3]
```diff
@@ -29,17 +28,32 @@ def test_pylint():
...
+            fail_count += 1
+        else:
+            print("Passed")
```
Comment (devoxin) [inline run_tests.py]: Single quotes

File: run_tests.py [hunk 4]
```diff
@@ -29,17 +28,32 @@ def test_pylint():
...
+
+    print(f"Fails: {fail_count}")
```
Comment (devoxin) [inline run_tests.py]: Single quotes

File: run_tests.py [hunk 5]
```diff
@@ -29,17 +28,32 @@ def test_pylint():
...
+    sys.exit(fail_count)
```
Comment (devoxin) [inline run_tests.py]: `0` if success, `1` if any failed.

**Review by devoxin — CHANGES_REQUESTED** [review]

**Comment by TAG-Epic** [comment]
Done!

**Pull request merged**